plugins/module_utils/network/nxos/utils/utils.py

```diff
@@ -163,3 +164,23 @@ def get_logging_sevmap(invert=False):
         # since we still test with Python 2.6
         x = dict(map(reversed, iteritems(x)))
     return x
+
+
+def vlan_unparser(data):
+    """
+    Returns a generator object that yields lists of
+    consequtive vlans.
+    """
+    for _k, group in groupby(data, lambda t, c=count(): int(t) - next(c)):
+        yield list(group)
+
+
+def vlan_list_to_range(cmd):
```

Comment:
can we add this utils function in utils similar to https://github.com/ansible-collections/ansible.utils/pull/77?

Reply:
Do you mean a filter plugin that converts list of vlans to ranges?

Reply:
yes